client/components/date-range/use-shortcuts.ts

```diff
@@ -79,19 +71,37 @@ export const getShortcuts = createSelector(
 				period: DATERANGE_PERIOD.DAY,
 			},
 			{
-				id: 'last_3_months',
-				label: translate( 'Last 90 Days' ),
-				startDate: siteToday.clone().subtract( 89, 'days' ).format( DATE_FORMAT ),
+				id: 'month_to_date',
+				label: translate( 'Month to date' ),
+				startDate: siteToday.clone().startOf( 'month' ).format( DATE_FORMAT ),
 				endDate: siteTodayStr,
-				period: DATERANGE_PERIOD.WEEK,
+				period: DATERANGE_PERIOD.DAY,
 			},
 			{
-				id: 'last_year',
-				label: translate( 'Last Year' ),
-				startDate: siteToday.clone().subtract( 364, 'days' ).format( DATE_FORMAT ),
+				id: 'last_12_months',
+				label: translate( 'Last 12 months' ),
+				startDate: siteToday
+					.clone()
+					.startOf( 'month' )
+					.subtract( 11, 'months' )
+					.format( DATE_FORMAT ),
 				endDate: siteTodayStr,
 				period: DATERANGE_PERIOD.MONTH,
 			},
+			{
+				id: 'year_to_date',
+				label: translate( 'Year to date' ),
+				startDate: siteToday.clone().startOf( 'year' ).format( DATE_FORMAT ),
```

Comment:
`Year to date` would be the same range as `Month to date` when today is the year's first month.

Reply:
Could filter those items whose startDate and endDate are duplicated at the stage? Otherwise it'll look too buggy to ship.

Reply:
In follow up PRs, we could then probably test the id of the shortcuts before matching the dates.

Reply:
Maybe initially we could just hide the YTD option? Then we can follow up with any necessary fixes to allow it to be properly selected. As it currently stands, attempting to select it results in the MTD shortcut being highlighted and applied.

I do think it's worth keeping as a visible option when the two periods match though, as we store that as a persistent view state.

Reply:
Kevin's suggest sound pretty pragmatic 👍

Reply:
Sounds good to me!